pkgs/by-name/ne/neovim-qt-unwrapped/package.nix

```diff
@@ -1,6 +1,6 @@
-{ lib, mkDerivation, fetchFromGitHub, cmake, doxygen, msgpack, neovim, python3Packages, qtbase, qtsvg }:
+{ lib, libsForQt5, fetchFromGitHub, cmake, doxygen, msgpack, neovim, python3Packages }:
 
-mkDerivation rec {
+libsForQt5.mkDerivation rec {
```

Comment:
Use stdenv.mkDerivation instead.

Reply:
I mimicked what was done at pkgs/by-name/qd/qdirstat/package.nix

Reply:
The message points to a wrapQtAppsHook missing at nativeBuildInputs.

Reply:
is this better ?